### PR TITLE
fix "Downloaded photo is not appearing in the photo album" in android

### PR DESCRIPTION
**Pull request by aimane-chnaif**
### Details
To export file to external storage, we need to follow https://github.com/RonRadtke/react-native-blob-util#copytomediastore.
After export, remove temp file from internal storage.
All files are downloaded in `/Download/Expensify` folder.
All media (images, photos) can be seen in native Gallery app.

### Fixed Issues
$ https://github.com/Expensify/App/issues/11101
PROPOSAL: https://github.com/Expensify/App/issues/11101#issuecomment-1283271264

### Tests
1. Login with any account in android app
2. Send or receive any attachment in any chat
3. Open attachment detail
4. Download
5. Open native Files app and locate `/Download/Expensify`
6. Verify that downloaded file appears in that folder.
For media (images or videos) only:
7.  Open native Gallery app
8. Verify that downloaded media appears in list

- [x] Verify that no errors appear in the JS console

### Offline tests
Download not working in offline mode

### QA Steps
1. Login with any account in android app
2. Send or receive any attachment in any chat
3. Open attachment detail
4. Download
5. Open native Files app and locate `/Download/Expensify`
6. Verify that downloaded file appears in that folder.
For media (images or videos) only:
7.  Open native Gallery app
8. Verify that downloaded media appears in list

- [ ] Verify that no errors appear in the JS console

### PR Author Checklist
- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

https://user-images.githubusercontent.com/96077027/225727012-73b2b7e4-2909-423f-958a-c63ef886b73b.mov

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://user-images.githubusercontent.com/96077027/225727071-3cb0ce5b-f41a-45ce-b905-8c975a1d8d46.mp4

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://user-images.githubusercontent.com/96077027/225727109-e6e5f36d-f463-4b0d-8e54-275ed7625225.mp4

</details>

<details>
<summary>Desktop</summary>

https://user-images.githubusercontent.com/96077027/225727154-31e087c1-ea8b-496a-86f0-6bc5586f2a70.mov

</details>

<details>
<summary>iOS</summary>

https://user-images.githubusercontent.com/96077027/225727220-38ea43c0-e8be-4213-8e37-ddf6a675aff4.mp4

</details>

<details>
<summary>Android</summary>

https://user-images.githubusercontent.com/96077027/225727257-90b00d25-6e65-441c-832e-d3aeb9c54784.mp4

</details>

### CODE REVIEW

File: src/libs/fileDownload/index.android.js
Comment (parasharrajat) [inline src/libs/fileDownload/index.android.js:69]: Can you please chain this to the outer promise instead of nested then and catch?

**Review by parasharrajat — CHANGES_REQUESTED** [review]

File: src/libs/fileDownload/index.android.js
Comment (parasharrajat) [inline src/libs/fileDownload/index.android.js:68]: let's swap these lines.

**Review by parasharrajat — CHANGES_REQUESTED** [review]
Although it is working fine, there is an issue.

BUG: Android: Clicking on the download notification throws error `media not found`. 

Steps: 
1. Send any attachment (image) message. 
2. Click to open the attachment. 
3. Click :arrow_down: download icon. 
4. Open notification tray. 
5. Click on the download complete notification.

**Comment by parasharrajat** [comment]
> BTW, I suggest not to use android download manager.

Why? 

> I tried to download file on android whatsapp and it's not using download manager so our app can be the same.

Not a good reasoning.

I think it is useful and there is a motive behind using it. But if you think so, feel free to open a slack discussion and explain the problem and reason. Don't forget to share the link here. You can tag me. 

--- 




Interesting, I can reproduce. Try with an image. 

https://user-images.githubusercontent.com/24370807/198079613-7ba1e2cf-9a30-4520-b86d-66c2f3e7ea1f.mp4

**Comment by parasharrajat** [comment]
Sure.

**Comment by parasharrajat** [comment]
There is a similar problem on the device. The Logical reason is that we are deleting the file after moving to mediasource which was downloaded via the manager. 


> I think it is useful and there is a motive behind using it. But if you think so, feel free to open a slack discussion and explain the problem and reason. Don't forget to share the link here. You can tag me.

Or we have to find a fix.

**Comment by parasharrajat** [comment]
Any update?

**Comment by Julesssss** [comment]
> There is a similar problem on the device. The Logical reason is that we are deleting the file after moving to mediasource which was downloaded via the manager.

Pretty certain we'll need to find a fix for this. It should be as simple as [hiding the DownloadManager notification](https://developer.android.com/reference/android/app/DownloadManager.Request.html#setNotificationVisibility(int)).  Hopefully the 3rd party library exposes this to us and it should be simple to fix 🤞 -- else I guess we'll need a fork or feature request

**Comment by parasharrajat** [comment]
@Julesssss  Could you please create a new issue we want to implement the notification? 

For now, it seems many agree to disable the notification for moving this PR. @aimane-chnaif let's do that for now. Also, same for iOS.

**Comment by Julesssss** [comment]
To move forward we would need an alternative solution for the invalid image URI, else we agree with [this workaround](https://expensify.slack.com/archives/C01SKUP7QR0/p1668546357958409?thread_ts=1668541406.340949&cid=C01SKUP7QR0) and hold this while a custom notification is implemented.

**Comment by parasharrajat** [comment]
I don't have access to the slack discussion. Could you please post it here?  I am good with holding this.

**Comment by Julesssss** [comment]
@parasharrajat oops sorry, I pasted the wrong link. Updated.

**Comment by Julesssss** [comment]
> Ok, so go ahead with this for now?

How about holding this instead? I'm working on a project for reworking notifications and it will eventually expose a simple method of posting notifications like this.

**Comment by parasharrajat** [comment]
Please add `[HOLD]` in the PR title and drop a comment on the issue specifying the reason for the hold.

**Comment by Julesssss** [comment]
Okay, let's reverse [this decision](https://github.com/Expensify/App/pull/12033#issuecomment-1317079741) and continue to work on this PR, as the invalid directory issue is a problem TODAY. We can improve with the custom notification shortly after.

**Review by parasharrajat — CHANGES_REQUESTED** [review]
@aimane-chnaif Please merge the main into this and update the videos to the latest app.

Looks like you need to update the author checklist as well. Go to the https://github.com/Expensify/App/blob/c40148b5ddd82b54ddb40f7437d1bee22fbf5682/.github/PULL_REQUEST_TEMPLATE.md and copy the template and then update the description with that.

Try to post videos directly into Github.

**Review by parasharrajat — DISMISSED** [review]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

cc: @Julesssss 

:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by Julesssss — DISMISSED** [review]
The MediaCollection solution only appears to work for newer Android OS versions. On my older device (Android 8.0) the image isn't saved to public storage. 

Presumably, this is either because MediaStorage isn't supported on older Android SDKs, or because [we need to add additional permissions](https://developer.android.com/training/data-storage/shared/media#extra-permissions). 

_I'll add some screencasts as a comment below_

File: src/libs/fileDownload/index.android.js
Comment (Julesssss) [inline src/libs/fileDownload/index.android.js:48]: Can you remind me why? IIRC this wasn't an option with MediaStorage. Do we even need this addAndroidDownloads config still?

**Comment by Julesssss** [comment]
Here's screencasts for Android 8 and 13

**Android 13**
https://user-images.githubusercontent.com/10736861/226413086-7e49cc54-1c5d-4da0-a995-4eca9257085b.mp4

**Android 8 (doesn't show in photos or files)**
https://user-images.githubusercontent.com/10736861/226413126-64275473-2248-4c09-9d85-2cdf265d335b.mp4

File: src/libs/fileDownload/index.android.js
Comment (Julesssss) [reply src/libs/fileDownload/index.android.js:48]: I'm still seeing the notification, even with this flag turned off 😕

**Comment by Julesssss** [comment]
From the Google developer site: 

> Downloaded files, which are stored in the Download/ directory. On devices that run Android 10 (API level 29) and higher, these files are stored in the [MediaStore.Downloads](https://developer.android.com/reference/android/provider/MediaStore.Downloads) table. **This table isn't available on Android 9 (API level 28) and lower.**

So I'd be interested to see how the [upstream lib maintainers](https://github.com/RonRadtke/react-native-blob-util/issues/170#issuecomment-1172943741) achieved this.

**Review by parasharrajat — APPROVED** [review]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

cc: @Julesssss 

:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by Julesssss — APPROVED** [review]
👍

**Comment by Julesssss** [comment]
I'm ignoring the checklist, it matches the provided link and looks good to me

**Pull request merged**
